packages/website/src/components/Playground.tsx

```diff
@@ -70,6 +73,19 @@ function Playground(): JSX.Element {
   const [position, setPosition] = useState<Monaco.Position | null>(null);
   const [activeTab, setTab] = useState<TabType>('code');
   const [showModal, setShowModal] = useState<TabType | false>(false);
+  const [editorDragging, setEditorDragging] = useState<boolean>(false);
+
+  useEffect(() => {
+    const handleMouseUp = (): void => {
+      setEditorDragging(false);
+    };
+
+    window.addEventListener('mouseup', handleMouseUp);
+
+    return (): void => {
+      window.removeEventListener('mouseup', handleMouseUp);
+    };
+  }, []);
```

Comment:
There's already logic in the `LoadedEditor` component that calls its `resize()` function on window resize. Now with this effect&state, there are _two_ places that could indicate to resize.

But I think this is making the component more complicated than it needs to be. Instead of having multiple places check individual DOM-based root causes for a resize (dragging the mouse, resizing the window, etc.), what about using a [`ResizeObserver`](https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver) inside `LoadedEditor` so it knows to resize whenever its DOM node changes? 

A couple benefits:
* We don't need to add more global listeners ad-hoc
* It'll know to resize on any change, so it'll keep resizing nicely as you move the dragging mouse around

We can try this as a followup IMO, it doesn't need to block this PR. But it's very much a nice to have 😄.

Reply:
@JoshuaKGoldberg Yes! This is my first time hearing about `ResizeObserver`, super cool.

Your solution works perfectly, you just needed to call `sandboxInstance.editor.getContainerDomNode()` instead of `sandboxInstance.editor.getDomNode()` 😄.